dom-event-delegation/main.js

```diff
@@ -0,0 +1,16 @@
+const $taskList = document.querySelector('.task-list');
+$taskList.addEventListener('click', handleClick);
+
+function handleClick(event) {
+  console.log('the event is', event);
+  console.log('the event target it', event.target);
+  console.log('the event targets tag name is', event.target.tagName);
+
+  const nearElement = event.target.closest('.task-list-item');
```

Comment:
Don't forget the `$` before the variable name! You're storing a DOM element, so we should include the dollar sign